docs/help/specifications-support/multiple-servers.md

```diff
@@ -0,0 +1,44 @@
+# Multiple Servers
+
+Both REST or Event-Driven APIs may run on different servers (production, staging, sandbox).
+
+And both [OpenAPI](https://spec.openapis.org/oas/v3.1.0#server-object) and [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/v2.6.0#serverObject) provide field `servers`, where more than one server can be defined.
+
+With multiple servers support, it’s now possible to render these different servers in documentation.
+
+## Setting multiple servers
+
+Until now, only first server was extracted and displayed.
+And maybe, as documentation owner you'd like to keep this behavior, and not expose your sandbox server url in your API documentation.
+
+To activate multiple servers on your documentation, there is a field to check on setting page:
+
+![multiple servers setting](/files/help/multiple-servers--setting.png)
+
+
+## Render multiple servers
+
+When this setting is activated, and when your specification contains several servers, then your documentation will display a list of all servers,
+accessible by this url: `${documentation_url}/servers`
+
+By default, first server is used to generate all curl examples on your documentation.
+
+You can select a different server from this selection component:
+
+![multiple servers component](/files/help/multiple-servers--component.png)
+
+For REST API documentation, url of selected server is used to generate curl example.
+For Event-Driven API documentation, no curl request, but protocol and protocol versions are updated in function of the selected server.
+
+By sharing current url of your documentation, all visitors with this link will see documentation with the selected server,
+thanks to query parameter `server_id`.
+
+:::info
+For REST API, alternative servers can be defined for every [OpenAPI Operation Object](https://spec.openapis.org/oas/v3.1.0#operation-object):
+
+> An alternative server array to service this operation. If an alternative server object is specified at the Path Item Object or Root level, it will be overridden by this value.
```

Comment:
Not sure what you're trying to say in the first sentence.

Reply:
🙈  Not my fault, it's a quote from https://spec.openapis.org/oas/v3.1.0#operation-object

![image](https://github.com/bump-sh/docs/assets/32060402/e6f86e3b-f5c7-41e8-b8c0-1636617dd0bc)

I can be more explicit with

> `operation.servers`: An alternative server array..

Reply:
Since we're describing a feature instead of defining a table, it would be better to have a normal sentence here.